archive/a/abap/README.md

```diff
@@ -0,0 +1,17 @@
+# Sample Programs in ABAP
+
+Welcome to Sample Programs in ABAP!
+
+## Sample Programs
+
+- [Hello World in ABAP](https://en.wikipedia.org/wiki/ABAP#Hello_world)
```

Comment:
Actually! One more change. Can you create an issue like #33 and link it here instead of the wiki article? Then, can you link the wiki article indented below like:

`    - Borrowed from [wikipedia](link)`

Reply:
Done. Source to Wikipedia does not have to be done anymore in my opinion.